lib/resque/plugins/timed_round_robin/timed_round_robin.rb

```diff
@@ -1,3 +1,5 @@
+require 'resque/plugins/timed_round_robin/configuration'
```

Comment:
I think you could put this require with the others here: https://github.com/KennaSecurity/resque-timed-round-robin/blob/master/lib/resque-timed-round-robin.rb